### PR TITLE
Add sensors test

### PR DESCRIPTION
**Pull request by jleveque**
- Also some minor cleanup/refactoring

### CODE REVIEW

File: ansible/roles/sonic-common/tasks/sensors_check.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+- name: Get platform monitor docker name
+  shell: docker ps -a --format '{{'{{'}}.Image{{'}} {{'}}.Names{{'}}'}}' | grep 'platform' | awk '{print $2}'
+  register: pmon_ps
+
+- set_fact: 
+    ansible_python_interpreter: "docker exec -i {{ pmon_ps.stdout }} python"
+
+- name: Gather sensors
+  sensors_facts: checks={{ sensors_checks[minigraph_hwsku] }}
```
Comment (pavel-shirshov) [inline ansible/roles/sonic-common/tasks/sensors_check.yml]: Please add library sensors_facts file

Comment (jleveque) [inline ansible/roles/sonic-common/tasks/sensors_check.yml]: sensors_facts.py is already committed.

Comment (pavel-shirshov) [inline ansible/roles/sonic-common/tasks/sensors_check.yml]: It's already in the repo

**Comment by lguohan** [comment]
can you generate pull against https://github.com/maggiemsft/sonic-mgmt/commits/masun/try_test_sync

**Comment by jleveque** [comment]
@lguohan: So to confirm, you would like me to merge this into Maggie's branch rather than master?

**Comment by jleveque** [comment]
Closing this PR, as it is now included as a part of PR https://github.com/Azure/sonic-mgmt/pull/183.

**Pull request closed without merging**